source/agora/common/DNS.d

```diff
@@ -911,13 +913,90 @@ public struct Domain
     /// The domain name
     public const(char)[] value = ".";
 
-    /// Creates domain name with always trailing root
-    public this (scope inout const(char)[] v) @trusted inout pure nothrow
+    @disable this () @safe pure nothrow @nogc;
+    @disable this (inout(char)[]) inout @safe pure nothrow @nogc;
```

Comment:
Default `ctor` would be enough

Reply:
I will double check, but I think it wasn't, or that it created nonsensical errors.

Reply:
Now I remember. It will give you an error like this:
```
onlineapp.d(7): Error: constructor `onlineapp.Container.this()` is not callable using argument types `(typeof(null))`
onlineapp.d(7):        expected 0 argument(s), not 1
```

Instead of:
```
onlineapp.d(8): Error: constructor `onlineapp.Container.this` cannot be used because it is annotated with `@disable`
```